### PR TITLE
SRCH-19 SRCH-20 remove Nutshell code

### PR DESCRIPTION
**Pull request by MothOnMars**
- add log lines for user & affiliate changes

### CODE REVIEW

**Comment by MothOnMars** [comment]
@peggles2 , the changes in this commit are almost all subtractive. However, I confirmed with Dawn that some of the notes we used to send to Nutshell still need to be logged, so you'll see some tweaks to add logging.

**Comment by MothOnMars** [comment]
P.S. The `nutshell_id` columns will be dropped in a subsequent commit.

File: app/models/user.rb [hunk 1]
```diff
@@ -215,27 +201,21 @@ def set_default_flags
...
   def audit_trail_user_removed(site, source)
-    add_nutshell_note_for_user('removed', 'from', site, source)
+    log_note_for_user('removed', 'from', site, source)
   end
 
-  def add_nutshell_note_for_user(added_or_removed, to_or_from, site, source)
-    note = "#{source} #{added_or_removed} @[Contacts:#{self.nutshell_id}], #{self.email} #{to_or_from} @[Leads:#{site.nutshell_id}] #{site.display_name} [#{site.name}]."
-    NutshellAdapter.new.new_note(self, note)
+  def log_note_for_user(added_or_removed, to_or_from, site, source)
+    note = "#{source} #{added_or_removed} User #{id}, #{email}, #{to_or_from} Affiliate #{site.id}, #{site.display_name} [#{site.name}]."
```
Comment (peggles2) [inline app/models/user.rb]: Maybe we can break this long line into 2 or 3 lines? its very long.

File: lib/tasks/user.rake [hunk 2]
```diff
@@ -6,7 +6,8 @@ namespace :usasearch do
         user.set_approval_status_to_not_approved
         user.save!
         Emailer.user_approval_removed(user).deliver_now
-        NutshellAdapter.new.new_note(user, 'This user is no longer associated with any sites, so their approval status has been set to "not_approved".')
+        note = "User #{user.id}, #{user.email}, is no longer associated with any sites, so their approval status has been set to \"not_approved\"."
```
Comment (peggles2) [inline lib/tasks/user.rake]: Break this into multiple lines.

File: spec/lib/tasks/user_spec.rb [hunk 3]
```diff
@@ -28,19 +26,19 @@
...
+      expect(Emailer).to receive(:user_approval_removed).with(user).and_return emailer
       expect(emailer).to receive(:deliver_now)
       @rake[task_name].invoke
     end
 
-    it 'creates a Nutshell note' do
-      expected_message = 'This user is no longer associated with any sites, so their approval status has been set to "not_approved".'
-      expect(nutshell_adapter).to receive(:new_note).with(users(:affiliate_manager_with_no_affiliates), expected_message)
+    it 'logs the change' do
+      expected_message = "User #{user.id}, #{user.email}, is no longer associated with any sites, so their approval status has been set to \"not_approved\"."
```
Comment (peggles2) [inline spec/lib/tasks/user_spec.rb]: May want to break this one to multiple lines too. .

File: spec/models/user_spec.rb [hunk 4]
```diff
@@ -635,8 +578,9 @@
     subject(:remove_from_affiliate) { user.remove_from_affiliate(site, 'Someone') }
 
     before do
-      expect(adapter).to receive(:push_site).with(site)
-      expect(adapter).to receive(:new_note).with(user, "Someone removed @[Contacts:1001], affiliate_manager@fixtures.org from @[Leads:99] NPS Site [nps.gov].")
+      expect(Rails.logger).to receive(:info).with(
+        "Someone removed User #{user.id}, affiliate_manager@fixtures.org, from Affiliate #{site.id}, NPS Site [nps.gov]."
+      )
```
Comment (peggles2) [inline spec/models/user_spec.rb]: same here.

File: spec/models/user_spec.rb [hunk 5]
```diff
@@ -568,58 +553,16 @@
...
     let(:site) { affiliates(:another_affiliate) }
 
     subject(:add_to_affiliate) { user.add_to_affiliate(site, 'Someone') }
 
     before do
-      site.update_attribute(:nutshell_id, 100)
-      expect(adapter).to receive(:push_site).with(site)
-      expect(adapter).to receive(:new_note).with(user, "Someone added @[Contacts:1001], affiliate_manager@fixtures.org to @[Leads:100] Another Gov Site [another.gov].")
+      expect(Rails.logger).to receive(:info).with(
+        "Someone added User #{user.id}, affiliate_manager@fixtures.org, to Affiliate #{site.id}, Another Gov Site [another.gov]."
```
Comment (peggles2) [inline spec/models/user_spec.rb]: And here

File: app/models/user.rb (hunk 1 above)
Comment (MothOnMars) [reply app/models/user.rb]: can do 👍

**Pull request merged**